presto-main/src/main/java/com/facebook/presto/server/ServerInfoResource.java

```diff
@@ -69,7 +71,7 @@ public ServerInfoResource(NodeVersion nodeVersion, NodeInfo nodeInfo, ServerConf
     @Produces(APPLICATION_JSON)
     public ServerInfo getInfo()
     {
-        boolean starting = !catalogStore.areCatalogsLoaded();
+        boolean starting = resourceManager ? true : !catalogStore.areCatalogsLoaded();
```

Comment:
```suggestion
        boolean starting = !resourceManager && !catalogStore.areCatalogsLoaded();
```

Reply:
By the way, why have this be true for RM?  Should it not be false?

Reply:
Not sure how the flag is been used, but if the server is up thought this has to be true. Do you have more context on and if we should keep this flag false or true for RM?

Reply:
I think `starting` indicates some sort of initialization is occurring.  There's no concept of that in RM right now, so my sense is it should always be `false` until that concept is developed and fully expressed in the code.